code42_connector.py

```diff
@@ -64,8 +61,11 @@ def _handle_test_connectivity(self, param):
         action_result = self.add_action_result(ActionResult(dict(param)))
         self.save_progress("Connecting to endpoint")
 
-        response = self._client.users.get_current()
-        util.print_response(response)
+        try:
+            self._client.users.get_current()
+        except Exception as e:
+            exception_message = e.args[0].strip()
```

Comment:
What if an exception occurs here that has no args?

Reply:
Example: 

```
class MyErr(Exception):
"""Test"""

e = MyErr()

# e.args is an empty tuple
```

Reply:
Ahh good catch. Fixed